sys/include/net/conn.h

```diff
@@ -12,19 +12,55 @@
  * @brief       Provides a minimal common API for applications to connect to the
  *              different network stacks.
  *
- * This module provides a minimal common API for applications to connect over different network
- * stacks. For each network stack there is supposed to be at least one connection type
- * implementation. Note that this definition gives no restriction on how a connection type should be
- * structured for simplicity and modularity. As a result, it can't give any guarantee that an
- * implementation keeps them compatible to each other. For example, an implementation might allow,
- * that a UDP receive function is called on a raw IPv6 connection object and even choose to do so
- * for valid reasons (e.g. code size), but this definition does not impose this on the
- * implementation. Currently there are the following option types defined:
+ * About
+ * =====
+ *
+* +---------------+
+ * |  Application  |
+ * +---------------+
+ *         ^
+ *         |
+ *         v
+ *       conn
+ *         ^
+ *         |
+ *         v
+ * +---------------+
+ * | Network Stack |
+ * +---------------+
```

Comment:
Where did I leave whitespaces? o.o sublimetext should usually prevent me from being so evil :D